src/main/java/org/galatea/starter/service/IexService.java

```diff
@@ -23,27 +24,43 @@
 
 
   /**
-   * Get all stock symbols from IEX.
+   * Exposes an endpoint to get all of the symbols available on IEX.
    *
-   * @return a list of all Stock Symbols from IEX.
+   * @param apiKey api token.
+   * @return a list of all IexStockSymbols.
    */
-  public List<IexSymbol> getAllSymbols() {
-    return iexClient.getAllSymbols();
+  public List<IexSymbol> getAllSymbols(final String apiKey) {
+    return iexClient.getAllSymbols(apiKey);
   }
 
   /**
-   * Get the last traded price for each Symbol that is passed in.
+   * Get the last traded price for each of the symbols passed in.
    *
-   * @param symbols the list of symbols to get a last traded price for.
-   * @return a list of last traded price objects for each Symbol that is passed in.
+   * @param apiKey api token.
+   * @param symbols list of symbols to get last traded price for.
+   * @return a List of IexLastTradedPrice objects for the given symbols.
    */
-  public List<IexLastTradedPrice> getLastTradedPriceForSymbols(final List<String> symbols) {
+  public List<IexLastTradedPrice> getLastTradedPriceForSymbols(final String apiKey,
+      final List<String> symbols) {
     if (CollectionUtils.isEmpty(symbols)) {
       return Collections.emptyList();
```

Comment:
Have we specified anywhere that this is the expected behavior when passing in an empty list of symbols?

Reply:
this function was already there and is not included in this task